### PR TITLE
Added new sections - services, team, advertising.

### PR DESCRIPTION
**Pull request by OlegAdamovich**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,5 +40,110 @@
...
+            <div class="team__item-content">
+                <div class="team__item-title">
+                    Lorem ipsum dolor sit amet
+                </div>
+                <div class="team__item-text">
+                    Sed egestas, ante et vulputate volutpat, eros pede semper est, vitae luctus metus libero eu augue. Morbi
+                    purus libero, faucibus adipiscing, commodo quis, gravida id, est. Sed lectus tpat, lacus a ultrices
+                    sagittis, mi neque euismod dui, eu pulvinar nunc sapien ornare nisl. Phasellus pede arcu, dapibus eu
+                </div>
+                <button class="button button_team">More</button>
```
Comment (mhryshkin) [inline index.html:118]: `button_team"`?

File: index.html [hunk 2]
```diff
@@ -40,5 +40,110 @@
...
+            </div>
+        </div>
+        <div class="team__item x">
```
Comment (mhryshkin) [inline index.html:121]: @OlegAdamovich Can we rename `x` to something more meaningful? Or, if you have no idea how to name it, maybe you need it at all?

**Comment by mhryshkin** [comment]
@OlegAdamovich I like how you use BEM. I see that you are proficient in usage `div` and `section` tags, I want you to go through the document and use more semantic elements(`p`, `h(2,3,4)`, `article`) in the places they might be used in your opinion.

File: index.html (hunk 2 above)
Comment (OlegAdamovich) [reply index.html:121]: It was a draft. Removed.

File: index.html (hunk 1 above)
Comment (OlegAdamovich) [reply index.html:118]: Renamed to `button_small`.

**Review by mhryshkin — APPROVED** [review]

**Pull request merged**